boards/rpi-pico/doc.txt

```diff
@@ -0,0 +1,138 @@
+since the RP2040 contains no internal flash.
+
+| MCU        | RP2040                                                       |
+|:-----------|:------------------------------------------------------------ |
+| Family     | (2x) ARM Cortex-M0+                                          |
+| Vendor     | Raspberry Pi                                                 |
+| RAM        | 264 KiB                                                      |
+| Flash      | 2 MiB (up to 16 MiB)                                         |
+| Frequency  | up to 133 MHz                                                |
+| FPU        | no                                                           |
+| PIOs       | 8                                                            |
+| Timers     | 1 x 64-bit                                                   |
+| ADCs       | 1x 12-bit (4 channels + temperature sensor)                  |
+| UARTs      | 2                                                            |
+| SPIs       | 2                                                            |
+| I2Cs       | 2                                                            |
+| RTCs       | 1                                                            |
+| USBs       | 1 (USB 2.0)                                                  |
+| Watchdog   | 1                                                            |
+| SSI/QSPI   | 1 (connected to flash, with XIP support)                     |
+| Vcc        | 1.62V - 3.63V                                                |
+| Datasheet  | [Datasheet](https://www.raspberrypi.org/homepage-9df4b/static/rp2040@2x-d1b9dae9345ad2bd15a23c6a567edb5c.jpg) |
+
+### User Interface
+
+1 button (also used for boot selection) and 1 LED:
+
+| Device | PIN              |
+|:------ |:---------------- |
+| LED0   | 25               |
+| SW0    | QSPI_SS_N (*)    |
+
+(*) Since the switch is connected to the chip-select pin of the QSPI interface the flash chip RIOT
+is running from via XIP, the switch is difficult to read out from software. This is currently not
+supported.
+
+### Pinout
+
+![Pinout Diagram of RPi Pico](https://projects-static.raspberrypi.org/projects/getting-started-with-the-pico/f009ad94826c2f0cd7573a295897e76955301096/en/images/Pico-R3-Pinout.png)
+
+## Flashing the Board
+
+### Flashing the Board Using OpenOCD
+
+Currently (June 2021), only two methods for debugging via OpenOCD are supported:
+
+1. Using a bit-banging low-level adapter, e.g. via the GPIOs of a Raspberry Pi 4B
+2. Using a virtual CMSIS-DAP adapter provided by the second CPU core via
+   https://github.com/majbthrd/pico-debug
+
+Since option 2 requires no additional hardware, this is currently the default. However, you need to
+first "flash" the gimme-cache variant of [pico-debug](https://github.com/majbthrd/pico-debug)
+into RAM using the UF2 bootloader. This will result in the Raspberry Pi Pico showing up as
+CMSIS-DAP debugger. Afterwards run:
```

Comment:
How is this done?
Or will the build system take care of that?

Reply:
The pico-debug .uf2 firmware needs to be copied into the virtual FAT flash drive. Upon unmounting that, the virtual flash drive will disappear and a CMSIS debugger will pop up. I could extend the documentation in this regard a bit, but I'd personally rather just keep the reference to pico-debug and let users read the upstream documentation.

This is not handled by the build system.

Reply:
OK, the upstream docu sucks in this regard. I'll add a sentence here.